### PR TITLE
Raise `psr/simple-cache: ^2.0|^3.0`, set psalm to 1.

### PR DESCRIPTION
**Pull request by terabytesoftw**
| Q             | A
| ------------- | ---
| Is bugfix?    | ❌
| New feature?  | ✔️
| Breaks BC?    | ❌

### CODE REVIEW

File: composer.json [hunk 1]
```diff
@@ -24,8 +24,9 @@
         "php": "^8.0",
         "ext-pdo": "*",
         "psr/log": "^2.0|^3.0",
-        "psr/simple-cache": "^1.0.1",
+        "psr/simple-cache": "^2.0|^3.0",
         "yiisoft/db": "^3.0@dev",
+        "yiisoft/db-sqlite": "3.0.x-dev",
```
Comment (vjik) [inline composer.json:29]: Seems, this package don't used.

Comment (samdark) [inline composer.json:29]: Looks wrong. Concreate DB driver should not be in require.

Comment (terabytesoftw) [reply composer.json]: fix error typo done :)

Comment (terabytesoftw) [reply composer.json]: fix error typo done :)

**Comment by terabytesoftw** [comment]
the tests mssql fail https://[pecl.php.net](https://pecl.php.net/)/ its down.

**Review by samdark — APPROVED** [review]

**Pull request merged**